sdk/communication/azure-communication-mediacomposition/CHANGELOG.md

```diff
@@ -0,0 +1,6 @@
+# Release History
+
+## 1.0.0b1 (Unreleased)
```

Comment:
I am not familiar with the python version format, but other languages set the version as `1.0.0-beta.1`

Reply:
For Python, the versioning format is a bit different. For example, Identity changelog looks like this: 
![image](https://user-images.githubusercontent.com/19615595/141185186-73cf944a-7200-40b5-8b3e-d50dcfe60e5d.png)